fineract-provider/dependencies.gradle

```diff
@@ -105,11 +105,12 @@ dependencies {
     )
 
     implementation 'org.apache.bval:org.apache.bval.bundle'
+    compile 'io.rest-assured:rest-assured'
```

Comment:
This line doesn't seem right, to me... we only want `rest-assured` to be _test_ scoped dependency, not a runtime dependency (i.e. it should never be part of the WAR) - agreed? Would you like to remove it? I doubt it will help with that (strange!) `NoClassDefFoundError`, but IMHO it's still worth a try, as a first step.

Reply:
I was trying to get rid NoClassDefFoundError that's why I placed it there.